subsys/bluetooth/controller/ll_sw/ull_central_iso.c

```diff
@@ -54,6 +54,24 @@
 #include "hal/debug.h"
 
 #define SDU_MAX_DRIFT_PPM 100
+#define SUB_INTERVAL_MIN 400
+
+#define CIG_ID_MAX 0xFE
+#define CIS_COUNT_MAX 0x1F
+#define SDU_INTERVAL_MIN 0x0000FF
+#define SDU_INTERVAL_MAX 0x0FFFFF
+#define WORST_CASE_SCA_VALID_MASK 0x07
+#define PACKING_VALID_MASK 0x01
+#define FRAMING_VALID_MASK 0x01
+#define MAX_TRANSPORT_LATENCY_MIN (0x0005 * 1000)
+#define MAX_TRANSPORT_LATENCY_MAX (0x0FA0 * 1000)
+#define CIS_ID_VALID_MAX 0xEF
+#define MAX_SDU_VALID_MASK 0x0FFF
+#define PHY_VALID_MASK 0x07
+#define ISO_INTERVAL_MIN 0x0004
+#define ISO_INTERVAL_MAX 0x0C80
```

Comment:
Several of these are also defined by `iso.h` if you want to use any of those

Reply:
This be defined (moved) to `include/zephyr/bluetooth/hci.h`, and reused in host and controller makes sense.

Reply:
They can, yes, but it isn't something we've done before. They are similar to the min/max values in `gap.h`, and can be useful for applications. We rarely expect applications to include `hci.h`.

Reply:
> We rarely expect applications to include hci.h

This is not true; `l2cap.h`, `iso.h` and `audio.h` include `hci.h` 

`hci.h` includes `conn.h`  which is incorrectly added in 0f06c7d8, hence making the controller dependent on host data types. This needs correction.

> They are similar to the min/max values in gap.h

GAP is different as the Core Spec defines recommended values for parameters, hence these recommended min/max/defaults are defined as public interface parameter values (these are not HCI min/max values).

Here too an exception has been with the introduction of these lines in `gap.h` (that shall be moved/defined in `hci.h`):
https://github.com/zephyrproject-rtos/zephyr/blob/c638d6fd21fd972a98cf114b89477abe9e8ed55f/include/zephyr/bluetooth/gap.h#L475-L482

There may be other exception that need correction, to not define HCI min/max values in upper layers.

> Not sure - @cvinayak - what is the normal pattern in the controller? Do we prefer to pull in host defines?

@mtpr-ot no, we do not pull host defines inside the Controller implementation. HCI command and event parameters are be defined in `hci.h`, like below:

https://github.com/zephyrproject-rtos/zephyr/blob/c638d6fd21fd972a98cf114b89477abe9e8ed55f/include/zephyr/bluetooth/hci.h#L884-L886

In the scope of this PR, let us define/use values from `hci.h` in the Controller implementation.

Reply:
Fair points :) 

We should consider moving all the HCI defined min/max values to `hci.h`, and possibly either deprecate the duplicated ones in `gap.h` and `iso.h` or keep them but use the macros from `hci.h` as their values (e.g. `#define BT_GAP_PER_ADV_MIN_INTERVAL BT_HCI_LE_PER_ADV_INTERVAL_MIN`)

Reply:
> keep them but use the macros from `hci.h` as their values (e.g. `#define BT_GAP_PER_ADV_MIN_INTERVAL BT_HCI_LE_PER_ADV_INTERVAL_MIN`)

This will help avoid stable API change request/deprecation.

Lets plan to do this outside this PR.